src/components/UNSTABLE__Pagination/__tests__/UNSTABLE_Pagination.spec.js

```diff
@@ -0,0 +1,412 @@
+/**
+ * @file UNSTABLE pagination tests.
+ * @copyright IBM Security 2019
+ */
+
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import { UNSTABLE__Pagination, PageSelector } from '../../..';
+
+import { namespace } from '../Pagination';
+
+describe('UNSTABLE_Pagination', () => {
+  test('should have no Axe or DAP violations', async () => {
```

Comment:
Would it make sense to move the a11y tests into it's own `describe` block? I know previously we had blocks for rendering, events, etc. so I'm wondering if we should apply a similar approach here

Reply:
I started out with nested blocks but they cluttered up the file a lot. Personally I'd prefer to leave them out and keep things flat. 🤔 Plus I think at this point, add blocks would be just to see certain indented output in the CI `yarn test` output?

Reply:
I'm of 2 minds here - I feel like it's nice to have the grouping to visually separate the tests out, but admittedly it becomes difficult with larger files.

I wonder if there's an alternative approach to reducing the cognitive burden here... My only initial suggestion is to break them out into separate files, but perhaps that's overkill. What are your thoughts here?

Reply:
Yea the visual separation of the extra `describe` blocks isn't easy to distinguish due to the clutter begin/end of the block creates, to me. And at the same time, I don't suggest splitting these tests out into multiple files... it's 400-ish lines right now & a split would increase the cognitive load of having both open at the same time (updating/adding tests would take longer as well).

Again, to me this flat, single file of 400-ish lines was easier to write, update, and read without the clutter or file splitting. 👍 More and more I'm in the camp of less abstraction = easier.

Reply:
Though on the topic of multiple tests files, I am going to make a new file for a couple `PageSelector` tests, I think. At least to check for a namespace class, since that class is relied on so heavily for styles not breaking etc.

Reply:
True, I thought it might be overkill...

I agree! That absolutely makes sense to have a suite specifically for the `PageSelector` - this should reduce this file's length and ensure we're only covering the behavior and logic of the `Pagination` here, and help us identify bloat in the process.